service/src/test/java/eu/europa/ec/fisheries/uvms/user/service/arquillian/BuildUserServiceDeployment.java

```diff
@@ -0,0 +1,59 @@
+package eu.europa.ec.fisheries.uvms.user.service.arquillian;
+
+import org.eu.ingwar.tools.arquillian.extension.suite.annotations.ArquillianSuiteDeployment;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+import org.junit.Assert;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+@ArquillianSuiteDeployment
+public abstract class BuildUserServiceDeployment extends Assert {
```

Comment:
What is the purpose of this file, is there any Arquillian test? Keep all Arquillian-related artifacts in a separate commit in a separate branch in case we need them.